src/pvecontrol/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+def defaulter(resource: dict, keys, default):
```

Comment:
I think this should be moved to `pvecontrol.utils`.

IMO, we should limit code in files __init__.py